ppa/archive/templates/archive/snippets/results_list.html

```diff
@@ -0,0 +1,23 @@
+{% load static humanize ppa_tags %}
```

Comment:
Perhaps a template comment here to indicate this template can be rendered in the full list or displayed via ajax?